system-tests/lib/cre/workflow/workflow.go

```diff
@@ -58,7 +58,7 @@ func RegisterWithCRECLI(input cretypes.ManageWorkflowWithCRECLIInput) error {
 		workflowSecretsURL = input.ExistingWorkflow.SecretsURL
 	}
 
-	registerErr := libcrecli.DeployWorkflow(input.CRECLIAbsPath, workflowURL, workflowConfigURL, workflowSecretsURL, creCLIWorkflowSettingsFile)
+	registerErr := libcrecli.DeployWorkflow(input.CRECLIAbsPath, workflowURL, workflowConfigURL, workflowSecretsURL, creCLIWorkflowSettingsFile, nil)
```

Comment:
me like not passing `nil`, because you don't know what is `nil` unless you open the function's definition... how about having a constant along the lines of `WithoutS3Provider`?